src/applications/disability-benefits/all-claims/pages/terminallyIll.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+
+const { terminallyIll } = fullSchema.properties;
```

Comment:
Once you get the [`vets-json-schema` PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/344) updated, this will need to change too, along with the property name in `uiSchema` and `schema`.